jest.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-module.exports = {
-	extends: ['plugin:jest/recommended', 'plugin:jest/style'],
-};
+const jest = require('eslint-plugin-jest');
+
+module.exports = [{ ...jest.configs['flat/recommended'] }, { ...jest.configs['flat/style'] }];
```

Comment:
[suggestion] Minor. When seeing <https://github.com/stylelint/eslint-config-stylelint/pull/298/files?diff=split&w=1#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R10-R12>, it seems to be able simplify this file:

```suggestion
module.exports = [jest.configs['flat/recommended'], jest.configs['flat/style']];
];
```